### DISCUSSION TITLE
KISS Poetry Wiki

### DISCUSSION THREAD

**Discussion by zerosonesfun** [Zavy86/WikiDocs#113] [Show and tell]
Hello!

I love WIkiDocs! I have a thing for web software that doesn't need a database. It's just cool. I started a poetry community recently and it has a lot of hidden or semi-hidden features. A wiki was needed to ensure new members figured things out.

[https://kisspoetry.com/wiki/](https://kisspoetry.com/wiki/)

I'm also a sponsor of this project (small one-time donation).

Thank you for making this!
Billy

**Comment by Zavy86** [comment]
@zerosonesfun https://www.wikidocs.app/examples

**Comment by ics01** [comment]
website no longer available @Zavy86  :-(

**Reply by Zavy86** [reply]
updated thanks
